src/install/kubernetes/helm/install-helm.mdx

```diff
@@ -0,0 +1,121 @@
+
+    ```shell
+    helm upgrade --install newrelic-bundle newrelic/nri-bundle \
+    --namespace newrelic --create-namespace \
+    -f values-newrelic.yaml
+    ```
+    
+6. Run this command to check that pods are deploying and reaching a stable state:
+
+    ```shell
+    kubectl -n newrelic get pods -w
+    ```
+
+7.  If you're going to use a GKE AutoPilot cluster, your Helm command should include the following:
+
+    ```yaml
+    --set newrelic-infrastructure.controlPlane.enabled=false \
+    --set newrelic-infrastructure.kubelet.config.scheme=http \
+    --set newrelic-infrastructure.kubelet.config.port=10255 \
+    ```
+
+    [Fluent Bit](https://fluentbit.io/) can use a FileStore volume to prevent data loss or duplicated logs during Fluent Bit pod restarts or redeploys.  Google FileStore API needs to be enabled in your GKE project. Check this [document](https://support.google.com/googleapi/answer/6158841) to know more about enabling APIs. 
+
+    To use FileStore, add the following:
+
+    ```yaml
+    --set newrelic-logging.fluentBit.persistence.mode=persistentVolume
+    --set newrelic-logging.fluentBit.persistence.persistentVolume.storageClass=standard-rwx
+    ```
+
+    <Callout variant="important">
+    The use of FileStore will result in additional costs, as charged by Google. See your Google Cloud Admin or the [FileStore](https://cloud.google.com/filestore/pricing) doc for more info.
+    </Callout>
+
+    If you don't want to use FileStore, add the following:
+
+    * Set `"newrelic-logging.fluentBit.persistence.mode"` to `"none"`
+    * Remove `"newrelic-logging.fluentBit.persistence.persistentVolume.storageClass"`
+
+8. Decide if you want to install and configure the Prometheus endpoints to scrape. See [Install and configure the Prometheus agent on Kubernetes clusters](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/) for more info. We recommend to scrape all Prometheus endpoints except core Kubernetes system metrics. Kubernetes integration already collects core Kubernetes system metrics in other components. 
+
+9. Gather Log data. You can:
+
+    * Forward all logs with full enrichment. Full enrichment ensures that all Kubernetes metadata, including labels and annotations, are associated with each log record. See [Fluent Bit](https://docs.fluentbit.io/manual/installation/kubernetes#concepts) documentation for more info.
+
+    * Forward all logs with minimal enrichment (low data mode). See [New Relic Logging](/docs/kubernetes-pixie/kubernetes-integration/installation/reduce-ingest/#new-relic-logging) for more info.
+
+    We recommend you forward all logs with minimal enrichment. This is a low-data mode that excludes labels excludes labels and annotations from log records, but keeps a minimal set of Kubernetes metadata. This set includes `cluster_name`, `container_name`, `namespace_name`, and `pod_name`.
+
+
+You should see the following:
```

Comment:
The pods listed are visible after installing, and it is not just for Helm but for any installation method